### PR TITLE
Fix/uniqueness status code

### PR DESCRIPTION
**Pull request by kamicut**
This moves the status code from 500 to the 400 range.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -20,9 +22,24 @@
...
+
+- Node v6
+- Yarn
+- Local Postgres 9.4+ running on localhost (default settings: username postgres, no password, port 5432)
+
+**Steps**
+
+- Clone this repo
+- `yarn` to install dependencies
+- `npm test` to run the tests
```
Comment (drewbo) [inline README.md]: @kamicut you can also do `yarn test` for consistency but not a necessary change

File: package.json [hunk 2]
```diff
@@ -5,7 +5,10 @@
   "main": "index.js",
   "scripts": {
     "start": "nf start",
-    "test": "AUTH0_SECRET='secret' NODE_ENV='test' nyc ava test"
+    "test": "npm run test-up && npm run test-script && npm run test-down",
+    "test-script": "AUTH0_SECRET='secret' NODE_ENV='test' nyc ava test",
+    "test-up": ".build_scripts/create_database.sh",
+    "test-down": ".build_scripts/drop_database.sh"
```
Comment (drewbo) [inline package.json:11]: 😍

File: routes/indicators/create_indicator.js [hunk 3]
```diff
@@ -36,6 +36,9 @@ module.exports = [
           return res({id: ret[0]});
         })
         .catch(function (err) {
+          if (err.code === '23505') {
```
Comment (drewbo) [inline routes/indicators/create_indicator.js:40]: What does this error code correspond to?

**Comment by drewbo** [comment]
👍 left a few small comments

File: routes/indicators/create_indicator.js (hunk 3 above)
Comment (kamicut) [reply routes/indicators/create_indicator.js:40]: This error corresponds to unique key violation in the database, I can add a comment https://www.postgresql.org/docs/current/static/errcodes-appendix.html

File: README.md (hunk 1 above)
Comment (kamicut) [reply README.md]: @drewbo I couldn't get `yarn test` to work with the rest of the package scripts

**Pull request merged**